tests/integration/slinky_setup.go

```diff
@@ -362,7 +362,7 @@ func (s *SlinkyIntegrationSuite) AddCurrencyPairs(chain *cosmos.CosmosChain, aut
 				},
 			},
 			Paths: mmtypes.Paths{Paths: []mmtypes.Path{
-				mmtypes.Path{Operations: []mmtypes.Operation{
```

Comment:
This file got added somehow for linting? @aljo242

Reply:
just a formatting think i guess